reporter/tap.js

```diff
@@ -0,0 +1,77 @@
+(function() {
+
+function reporter( rep ) {
+	if ( rep === "tap" ) {
+		tapReporter();
+	}
+};
+
+reporter.print = function( val ) {
+	console.log( val );
+};
+
+QUnit.reporter = reporter;
+
+function tapReporter() {
+	var testCount = 0;
+	var failed = 0;
+
+	QUnit.begin( function() {
+		QUnit.reporter.print( "TAP version 13" );
```

Comment:
I wanted something I can patch/replace/inspect. Do you have any other suggestion?